### DISCUSSION TITLE
Static/customizable background / liveview only on button-press

### DISCUSSION THREAD

**Discussion by PixelAlarm** [photobooth-app/photobooth-app#170] [Ideas]
Hello,

first of all a big thanks for this photobooth-solution! I really like the approach.

It would be really nice to be able to set a custom background for the welcome page.
This would help to customize the look of the app. (Is more customizability - e. g. colors -planned in the future?)

For the background to be seen, it would be nice to show live view only after a button is pressed.

**Comment by mgineer85** [comment]
Hi,

there is a pull request open right now for bit more customization and change the color themes.

as to the background it is possible with not too much effort in the frontend.
The text on the liveview you can use pure html? How far could you get with that approach to add information instead of a background?

**Reply by PixelAlarm** [reply]
Hi,

this was very helpful.

With the change of the frontage text to the following inside the div, I have an overlay I can adjust to act as background and show live view only after button is pressed:
class="fixed-center text-h2 text-weight-bold text-center text-white" style="image"><img src="bg.jpg" alt="Background"

Looking forward to the color picker being merged :)
